vms/platformvm/block/executor/block.go

```diff
@@ -24,7 +24,9 @@ var (
 // Exported for testing in platformvm package.
 type Block struct {
 	block.Block
-	manager *manager
+	manager                  *manager
+	rejected                 bool
```

Comment:
Why are we adding this `rejected` bool?

Reply:
To report it as `choices.Rejected` in `Status()`

Reply:
Is this a behavior change from before? Previously, we didn't store a block being `rejected` but now we do, and report it in `Block.Status`

Reply:
Yes, this is a change. I can remove it but thought it'd be good to match the AVM